jest.config.js

```diff
@@ -16,12 +16,10 @@ module.exports = {
     'frontend/js',
     'frontend/js/app',
   ],
-  snapshotSerializers: [
-    'enzyme-to-json/serializer',
-  ],
-  setupFiles: [
+  setupFilesAfterEnv: [
```

Comment:
there's no need to set things before env, but now you need to import RTL after env, so I updated this.